components/arm/universalrobots/ur.go

```diff
@@ -107,6 +108,25 @@ type URArm struct {
 
 const waitBackgroundWorkersDur = 5 * time.Second
 
+// UpdateAction helps hinting the reconfiguration process on what strategy to use given a modified config.
+// See config.UpdateActionType for more information.
+func (ua *URArm) UpdateAction(c *config.Component) config.UpdateActionType {
+	newCfg, ok := c.ConvertedAttributes.(*AttrConfig)
+	if ok {
+		if ua.speed != newCfg.Speed {
+			ua.speed = newCfg.Speed
+		}
+		if ua.host != newCfg.Host {
```

Comment:
This will almost certainly fail terribly; if `host` changes it's probably worth doing the full reconfigure.